integration/helpers/resource.py

```diff
@@ -107,3 +114,50 @@ def create_bucket(bucket_name, region):
         s3_client = boto3.client("s3", region_name=region)
         location = {"LocationConstraint": region}
         s3_client.create_bucket(Bucket=bucket_name, CreateBucketConfiguration=location)
+
+
+def load_yaml(file_path):
+    """
+    Loads a yaml file
+
+    Parameters
+    ----------
+    file_path : Path
+        File path
+
+    Returns
+    -------
+    Object
+        Yaml object
+    """
+    with open(file_path) as f:
+        data = f.read()
+    return yaml.load(data, Loader=yaml.FullLoader)
+
+
+def should_exclude_test_in_region(exclude_resource):
```

Comment:
I would rename this to `current_region_does_not_support(service)` so that it reads like a sentence:

`@skipIf(current_region_does_not_support("ServerlessRepo"), ...)`

Reply:
Also should this accept a list instead of a single item?

Reply:
Good idea, updated.

Reply:
@mgrandis .. if it will support list, so shall it return True if all services in the list are excluded, or just any of them

Reply:
I would say any.
If the test relies on 2 services and only 1 is not available in a region it shouldn't run.